src/index.d.ts

```diff
@@ -1037,6 +1152,26 @@ declare namespace admin.remoteConfig {
     */
     rollback(versionNumber: string | number): Promise<RemoteConfigTemplate>;
 
+    /**
+    * Gets a list of Remote Config template versions that have been published, sorted in reverse 
+    * chronological order. Only the last 300 versions are stored.
+    * All versions that correspond to non-active Remote Config templates (i.e., all except the 
+    * template that is being fetched by clients) are also deleted if they are older than 90 days.
+    * 
+    * @param options Optional options object for getting a list of tempalte versions:
+    *    - {number} `pageSize` The maximum number of items to return per page.
```

Comment:
I don't think we do this anywhere else. Let TSDoc just link this to `ListVersionsOptions`.

Reply:
We did something similar for `publishTemplate` `options` parameter. I agree with you though that we have a lot more parameters here. Will just use a link here.